app/models/offender.rb

```diff
@@ -31,9 +33,34 @@ class Offender < ApplicationRecord
           inverse_of: :offender,
           dependent: :destroy
 
-  has_one :parole_record, foreign_key: :nomis_offender_id, inverse_of: :offender, dependent: :destroy
+  has_many :parole_records, foreign_key: :nomis_offender_id, inverse_of: :offender, dependent: :destroy
 
   has_one :calculated_early_allocation_status, foreign_key: :nomis_offender_id, inverse_of: :offender, dependent: :destroy
 
   has_many :victim_liaison_officers, foreign_key: :nomis_offender_id, inverse_of: :offender, dependent: :destroy
+
+  delegate :parsed_hearing_outcome, to: :parole_record
```

Comment:
I think actually the entire thing may be redundant now and a bit of a relic of an approach I was taking previously, as I believe I ended up refactoring `parsed_hearing_outcome` into two separate methods.
I'm hesitant to remove it as it stands in case I'm missing somewhere that it's relied on in a later ticket (the perils of daisy-chaining commits ey?), but I'll make a note to remove it if it's not used anywhere once everything is merged into MO-Parole